packages/styles/src/components/form-check.scss

```diff
@@ -49,10 +50,11 @@
     }
 
     @include utility-mx.focus-style-none();
+
     display: inline-flex;
     flex: 0 auto;
     appearance: none;
-    background: color.$white;
+    background: transparent;
```

Comment:
Checkboxes are not visible on dark background. This change was done for card-control